src/Uri.php

```diff
@@ -641,7 +641,7 @@ private function filterQuery($query)
     private function splitQueryValue($value)
     {
         $data = explode('=', $value, 2);
-        if (1 === count($data)) {
+        if (! isset($data[1])) {
```

Comment:
This check is not equivalent to the previous: you need to verify that the key `0` exists too

Reply:
@Ocramius `explode()` should always has key 0 ? https://3v4l.org/viNZT

Reply:
@Ocramius agree with @samsonasik

Reply:
Yup, misread